### PR TITLE
feat: add svelte preset

### PR DESCRIPTION
**Pull request by hendriksoares**

### CODE REVIEW

File: tests/e2e/svelte-static.test.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  }, TIMEOUT);
+
+  afterAll(async () => {
+    await projectStop(serverPort, EXAMPLE_PATH.replace('/examples/', ''));
+
+    await browser.close();
+  }, TIMEOUT);
+
+  test('Should render home page in "/" route', async () => {
+    await page.goto(`${localhostBaseUrl}/`);
```
Comment (jcbsfilho) [inline tests/e2e/svelte-static.test.js:39]: Before checking the content, it is necessary to wait for the render resolution.

```js
await page.goto(`${localhostBaseUrl}/`, {
    waitUntil: 'domcontentloaded',
});
await page.waitForSelector('h1');
const pageContent = await page.content();

```

File: tests/e2e/svelte-static.test.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    const pageContent = await page.content();
+
+    expect(pageContent).toContain('Welcome to SvelteKit');
+    expect(pageContent).toContain(
```
Comment (jcbsfilho) [inline tests/e2e/svelte-static.test.js:44]: Content contains html tags, please validate text or insert tags.

```js

expect(pageContent).toContain('kit.svelte.dev');

```

File: tests/e2e/svelte-static.test.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      'Visit kit.svelte.dev to read the documentation',
+    );
+  });
+
+  test('Should render edge page in "/edge" route', async () => {
```
Comment (jcbsfilho) [inline tests/e2e/svelte-static.test.js:49]: This route does not exist in the example

File: tests/e2e/svelte-static.test.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    await page.goto(`${localhostBaseUrl}/edge`);
+
+    const pageContent = await page.content();
+
+    expect(pageContent).toContain('Running in Edge.');
+  });
+
+  test('Should return correct asset', async () => {
+    await request
+      .get('/favicon.ico')
```
Comment (jcbsfilho) [inline tests/e2e/svelte-static.test.js:59]: The favicon is png.

```js
await request
      .get('/favicon.png')
      .expect(200)
      .expect('Content-Type', /image\/png/);

```

File: tests/e2e/svelte-static.test.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    const pageContent = await page.content();
+
+    expect(pageContent).toContain('Welcome to SvelteKit');
+    expect(pageContent).toContain(
+      'Visit kit.svelte.dev to read the documentation',
+    );
+  });
+
+  test('Should render edge page in "/edge" route', async () => {
+    await page.goto(`${localhostBaseUrl}/edge`);
```
Comment (jcbsfilho) [inline tests/e2e/svelte-static.test.js:50]: Please add wait

```js
await page.goto(`${localhostBaseUrl}/`, { waitUntil: 'domcontentloaded', });
```

**Review by jcbsfilho — CHANGES_REQUESTED** [review]

**Review by jotanarciso — APPROVED** [review]

**Review by enicio — APPROVED** [review]

**Pull request merged**